app-center-services/src/main/resources/locale/addon/appcenter_en.properties

```diff
@@ -43,21 +53,25 @@ appCenter.adminSetupForm.descriptionPlaceHolder=Enter a small description to exp
 appCenter.adminSetupForm.mandatory=Mandatory
 appCenter.adminSetupForm.active=Active
 appCenter.adminSetupForm.mobile=Mobile
+appCenter.adminSetupForm.table.switch.mobile.tooltip=Visibility in mobile
+appCenter.adminSetupForm.table.editApp.tooltip=Edit application
+appCenter.adminSetupForm.table.deleteApp.tooltip=Delete application
+appCenter.adminSetupForm.table.footer.text=Items per page
 appCenter.adminSetupForm.permissions=Permissions
 appCenter.adminSetupForm.permissionsPlaceHolder=Add permissions
 appCenter.adminSetupForm.helpPage=Help page
 appCenter.adminSetupForm.helpPagePlaceholder=Enter a valid URL
 appCenter.adminSetupForm.save=Save
 appCenter.adminSetupForm.cancel=Cancel
 appCenter.adminSetupForm.edit=Edit
-appCenter.adminSetupForm.delete=Remove
+appCenter.adminSetupForm.modal.delete=Delete
 appCenter.adminSetupForm.browse=Browse
 appCenter.adminSetupForm.error=Server error
 appCenter.adminSetupForm.unauthorized=The application already exists with the same name or url
-appCenter.adminSetupForm.confirmDelete=Do you want really to remove application {0} ? 
+appCenter.adminSetupForm.modal.confirmDelete=Do you really want to delete this application ? This process can not be undone.
 appCenter.adminSetupForm.noApp=No application
 appCenter.adminSetupList.application=Application
-appCenter.adminSetupList.picto=Pictogram
+appCenter.adminSetupList.avatar=Avatar
```

Comment:
Icon

Reply:
This is for the applications table's image column header.